README.md

```diff
@@ -1,66 +1,83 @@
+   $ cd lead-alerts-node
+   ```
 
-### Install Dependencies
+1. Next, open `.env` at the root of the project and update it with
+  values from your
+  [Twilio account](https://www.twilio.com/console)
+  and local configuration. You'll need to set
+  `TWILIO_AUTH_TOKEN`, `TWILIO_ACCOUNT_SID`, and `TWILIO_NUMBER`.
 
-Navigate to the project directory in your terminal and run:
+  For the `TWILIO_NUMBER` variable you'll need to provision a new number
+  in the
+  [Manage Numbers page](https://www.twilio.com/user/account/phone-numbers/incoming)
+  under your account.
 
-```bash
-npm install
-```
+  The `AGENT_NUMBER` variable represents the number alerts will be sent to. Please make sure you have allowed SMS to be sent to the Country this number belongs to on the [Global SMS Permissions page](https://www.twilio.com/console/sms/settings/geo-permissions). Also, if you are on a trial account, make sure you have verified this number on the [Verified Callers IDs page](https://www.twilio.com/console/phone-numbers/verified).
 
-This should install all of our project dependencies from npm into a local `node_modules` folder.
+  The phone numbers should be in
+  [E.164 format](https://www.twilio.com/help/faq/phone-numbers/how-do-i-format-phone-numbers-to-work-internationally).
 
-### Configuration
+  Run `source .env` to export the environment variables.
 
-Next, open `config.js` at the root of the project and update it with values from your environment and your [Twilio account](https://www.twilio.com/user/account/voice-messaging). You can either export these values as system environment variables (this is the default setup), or you can replace these values with hard-coded strings (be careful you don't commit them to git!).
+1. Navigate to the project directory in your terminal and run:
 
-### Running the Project
+  ```bash
+  $ npm install
+  ```
 
-To launch the application, you can use `node .` in the project's root directory. You might also consider using [nodemon](https://github.com/remy/nodemon) for this. It works just like the node command, but automatically restarts your application when you change any source code files.
+  This should install all of our project dependencies from npm into a local `node_modules` folder.
 
-```bash
-npm install -g nodemon
-nodemon .
-```
+1. To launch the application, you can use `node .` in the project's root directory. You might also consider using [nodemon](https://github.com/remy/nodemon) for this. It works just like the node command, but automatically restarts your application when you change any source code files.
+
+  ```bash
+  $ npm install -g nodemon
+  $ nodemon .
+  ```
 
 ### Docker local development
 
-First, install [Docker](https://www.docker.com/) and [docker-compose](https://docs.docker.com/compose/install/).
+1. Install [Docker](https://www.docker.com/) and [docker-compose](https://docs.docker.com/compose/install/).
+
+1. Make sure you performed step `2` on the previous section.
```

Comment:
What do you think about?

.. step **2** shown in the previous section.

I do prefer to use back ticks for highlighting code or something related.